mappings/net/minecraft/client/font/TextRenderer.mapping

```diff
@@ -50,30 +52,34 @@ CLASS net/minecraft/class_327 net/minecraft/client/font/TextRenderer
 		ARG 2 x
```

Comment:
Change that `str` above this line to `text` as well.

Reply:
Oh, good catch!

Should really have all of them changed.